packages/typora-deb/typora-deb.pacscript

```diff
@@ -0,0 +1,18 @@
+pkgname="typora-deb"
+gives="typora"
+pkgver="1.9.3"
+repology=("project: typora")
+depends=("libgtk-3-0" "libnss3")
+arch=('amd64')
+license=('custom:"Copyright (c) 2015 Abner Lee All Rights Reserved."')
```

Comment:
This has to change. It's not a valid license name.

Reply:
How so? This is proprietary payware software. It's not using any SPDX type licensing but their own.

Reply:
then you don't need to provide the license variable at all (and shouldn't). It is meant for licenses that are already commonly available to systems through the `spdx-licenses` package (which, now that I say this out loud, should be made a pacstall dependency for next version).